src/particle/converters/bimap.py

```diff
@@ -212,16 +248,16 @@ def __getitem__(self, value):
             raise MatchingIDNotFound(msg)
 
     def __iter__(self):
+        # type: () -> Iterator[str]
         return iter(self._map)
 
     def __repr__(self):
+        # type: () -> str
         return "<{self.__class__.__name__}({a}->{b}): {n} matches>".format(
             self=self, a=self.name_A, b=self.name_B, n=self.__len__()
         )
 
-    def __str__(self):
-        return repr(self)
-
```

Comment:
This method is needed for int subclasses, because we have to replace the existing one. Otherwise, though, this is exactly the default definition! :)

Reply:
Sure :-).